dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -246,5 +248,14 @@ private static boolean isConnected(final Connection conn) throws SQLException {
         return (conn != null && !conn.isClosed());
     }
 
+    private Map<Class<?>, TypeSpecifier> typeSpecifiers = new HashMap<Class<?>, TypeSpecifier>();
+
+    public TypeSpecifier getTypeSpecifier(Class<?> targetClass) {
+    	return typeSpecifiers.get(targetClass);
+    }
```

Comment:
I've made this change - to pass the `TypeSpecifier` map instead of the `DBEnvironment`.